spec/lib/solidus_kustomer/client_spec.rb

```diff
@@ -14,6 +14,32 @@
     end
   end
 
+  describe '#create_customer' do
+    it 'creates the customer' do
+      user = create(:user_with_addresses, id: 500)
+      kustomer = described_class.new(api_key: 'my_api_key')
+
+      VCR.use_cassette('create_customer') do
+        response = kustomer.create_customer(user)
+        expect(response['attributes']['externalId']).to eq(user.id.to_s)
+      end
+    end
+  end
+
+  describe '#update_customer' do
+    it 'updates the existing customer information on Kustomer' do
+      user = create(:user, kustomer_id: 'the_customer_UUID')
+      kustomer = described_class.new(api_key: 'my_api_key')
+
+      user.update(email: 'the_new_email@example.com')
```

Comment:
Always use bang methods when preparing records for a test. That way, if the preparation fails, your test fails loudly with a clear error message, rather than making your expectations fail.

Reply:
Noted for the future, thanks.